packages/main/src/themes/BusyIndicator.css

```diff
@@ -0,0 +1,96 @@
+:host(ui5-busy-indicator) span[data-sap-ui-wc-root] {
+	display: inline-block;
+	width: 100%;
+	height: 100%;
+}
+
+ui5-busy-indicator span [data-sap-ui-wc-root] {
+	display: inline-block;
+	width: 100%;
+	height: 100%;
+}
+
+.ui5-busy-indicator-wrapper {
```

Comment:
When you change the tag name from ui5-busy-indicator to ui5-busyindicator, don`t forget to rename the css selectors and css variables.